.github/workflows/build_main.yml

```diff
@@ -29,4 +29,4 @@ jobs:
     permissions:
       packages: write
     name: Run
-    uses: ./.github/workflows/backend.yml
+    uses: ./.github/notify_test_workflow.yml
```

Comment:
Hi @dailai , please do not change the workflow file. It should work fine.

Reply:
Yes, it was a mistake